src/reducers/fileTree.spec.tsx

```diff
@@ -774,7 +784,27 @@ describe(__filename, () => {
             messageMap,
             position: RelativePathPosition.next,
           }),
-        ).toEqual({ path: file1, uid: line1FirstUid });
+        ).toEqual(expect.objectContaining({ uid: line1FirstUid }));
+      });
+
+      it('returns the expected line number for a byLine message', () => {
+        // Expect the first message on line 1.
+        expect(
+          _getRelativeMessage({
+            currentMessageUid: global2,
```

Comment:
For future reference, it reduces test readability to rely on a [common setup routine](https://github.com/mozilla/addons-code-manager/blob/2fb604f38607f7fc4593adefb8139e81117ee9d5/src/reducers/fileTree.spec.tsx#L737-L748). It's hard for me to look at this test and understand what the inputs are. 

It's convenient to rely on common setup like this but you can still get the right amount of convenience by calling a setup helper explicitly at the top of every test with just a few lines of code.

This is not a change request, just a comment.

Reply:
I totally understand what you mean, and even thought about it while adding these new tests - the fact that the inputs aren't obvious from the test itself. I'll keep this in mind for the next time I consider doing this, or if we need to do more work on these test cases.